.github/workflows/release.yml

```diff
@@ -156,6 +156,19 @@ jobs:
             phylum-*.zip
             phylum-*.zip.minisig
 
+      - name: Trigger phylum-ci Docker image creation
+        # Don't trigger for pre-releases
+        if: ! contains(github.ref, 'rc')
+        # Reference: https://docs.github.com/en/rest/repos/repos#create-a-repository-dispatch-event
+        run: >
+          curl
+          -X POST
+          --fail-with-body
+          -H "Accept: application/vnd.github.v3+json"
+          -H "Authorization: token ${{ secrets.GH_RELEASE_PAT }}"
+          -d '{"event_type":"build-push-docker-images","client_payload":{"CLI_version":"$GITHUB_REF_NAME"}}'
+          https://api.github.com/repos/phylum-dev/phylum-ci/dispatches
```

Comment:
Can you indent the arguments without it breaking? Having it all on the same level as `curl` kinda makes it difficult to distinguish from multiple commands in the same script.

Reply:
Yes, it is possible. Some [test cases were added to a throwaway branch](https://github.com/phylum-dev/cli/blob/yaml_case/.github/workflows/test.yml#L76-L148) and [manually run in CI](https://github.com/phylum-dev/cli/runs/6884226132?check_suite_focus=true), yielding the following results:

* Simply indenting the lines does not work
* Adding trailing backslash line continuation characters works to indent the command line
  * It works for *both* folded (`>`) and literal (`|`) yaml block styles
* Using the `--no-progress-meter` option is helpful to reduce output noise
* The `--fail-with-body` option was added in `curl` v7.76.0
  * This version is not available in `ubuntu-latest`
  * This version is available in `ubuntu-22.04`

It comes down to a style choice in whether it looks better to have indentation *with* trailing backslashes or no indentation but also no trailing backslashes. The assumption is that indentation is still the preferred option even if it means adding a different kind of cognitive load to future readers of this code. @cd-work, Is that correct?

This review will be updated with those findings shortly.